ui/v2.5/src/components/List/Pagination.tsx

```diff
@@ -16,6 +16,11 @@ export const Pagination: React.FC<IPaginationProps> = ({
 }) => {
   const totalPages = Math.ceil(totalItems / itemsPerPage);
 
+  // Build the n-m/total string
+  const firstItemCount:number = Math.min((currentPage-1)*itemsPerPage+1, totalItems);
+  const lastItemCount:number = Math.min(firstItemCount+(itemsPerPage-1), totalItems);
+  const indexText:string = `${firstItemCount}-${lastItemCount}/${totalItems}`;
```

Comment:
You don't actually need to specify the type since the type is implied from the initialization.

Reply:
I know, my C days just give me anxiety regarding inferred types.